redis/k8s-redis/redis-configmap.tf

```diff
@@ -0,0 +1,22 @@
+data "template_file" "configmap" {
```

Comment:
Please add comments

Reply:
Let's have local names be more descriptive.  `configmap` is confusing in itself